tasks/us/census/acs.py

```diff
@@ -24,6 +24,14 @@
 from time import time
 LOGGER = get_logger(__name__)
 
+GEOGRAPHIES = ['state', 'county', 'census_tract', 'block_group',
+                        'puma', 'zcta5', 'school_district_elementary',
+                        'congressional_district',
+                        'school_district_secondary',
+                        'school_district_unified', 'cbsa', 'place']
+YEARS = ['2015', '2010']
+SAMPLES = ['5yr', '1yr']
```

Comment:
This does not cover all cases, we have:
-5 years: 2010, 2015
-1 year: 2010, **2014**, 2015